cdc/entry/mounter.go

```diff
@@ -405,9 +411,14 @@ func (m *mounterImpl) mountRowKVEntry(tableInfo *TableInfo, row *rowKVEntry) (*m
 		for _, col := range tableInfo.Columns {
 			_, ok := values[col.Name.O]
 			if !ok && tableInfo.IsColWritable(col) {
+				var flag model.ColumnFlagType
+				if col.Charset == "binary" {
+					flag.SetIsBinary()
+				}
```

Comment:
recommend encapsulating this logic into a function like
```golang
func transColumnFlag(col timodel.ColumnInfo) model.ColumnFlagType {
    // some logic
}
```
we can reuse it

Reply:
OK.